app/controllers/spree/admin/reports_controller_decorator.rb

```diff
@@ -53,17 +53,17 @@ def report_types
     }
   end
 
-  # Overide spree reports list.
+  # Override spree reports list.
   def index
     @reports = authorized_reports
     respond_with(@reports)
   end
 
-  # This action is short because we refactored it like bosses
+  # This action is short because we re-factored it like bosses
```

Comment:
I've always seen `refactor` and never `re-factor` in the computer science literature :thinking:

Reply:
I've seen both and thought that "re-factor" would be grammatically correct. It was probably the influence of my outdated spell checker that made me do this change. I'm happy to remove it.

Reply:
Looking at this again, now that we have made more of these methods short, we can probably delete the whole comment.